cmd/kubelet/app/options/options.go

```diff
@@ -256,14 +256,14 @@ func ValidateKubeletFlags(f *KubeletFlags) error {
 
 	unknownLabels := sets.NewString()
 	for k := range f.NodeLabels {
-		if isKubernetesLabel(k) && !kubeletapis.IsKubeletLabel(k) {
+		if isKubernetesLabel(k) && !v1.IsKubeletLabel(k) {
```

Comment:
keep the `IsKubeletLabel` helper method where it was... we don't typically put helper function in the k8s.io/api packages